.expeditor/scripts/chef-infra-client.spec

```diff
@@ -0,0 +1,89 @@
+# Disable any shell actions, replace them with simply 'true'
+%define __spec_prep_post true
+%define __spec_prep_pre true
+%define __spec_build_post true
+%define __spec_build_pre true
+%define __spec_install_post true
+%define __spec_install_pre true
+%define __spec_clean_post true
+%define __spec_clean_pre true
+
+# Use SHA256 checksums for all files
+%define _binary_filedigest_algorithm 8
+
+%define _binary_payload w1.xzdio
+
+# Disable creation of build-id links
+%define _build_id_links none
+
+Name:           chef-infra-client
+Version:        %{VERSION}
+Release:        1%{?dist}
+Summary:        The full stack of Chef Infra Client
+AutoReqProv: 	no
+BuildRoot: 	    %buildroot
+Prefix: 	    /
+Group: 		    default
+License:        Chef EULA
+URL: 		    https://www.chef.io
+Packager: 	    Chef Software, Inc. <maintainers@chef.io>
+Source0:        %{CHEF_INFRA_TAR}
+Source1:        %{CHEF_MIGRATE_TAR}
+
+%description
+The full stack of Chef Infra Client
+
+%prep
+# noop
+
+%build
+# noop
+
+%install
+# Create the installation directory for the migration tools
+#mkdir -p %{buildroot}/opt/chef
+mkdir -p %{buildroot}/opt/chef/{bin,bundle}
+
+# Untar the migration tools into /opt/chef/bin
+tar -xf %{SOURCE1} -C %{buildroot}/opt/chef/bin
+
+# Copy the chef infra tarball into /opt/chef/bundle
+cp %{SOURCE0} %{buildroot}/opt/chef/bundle/
+
+%files
+/opt/chef
+/opt/chef/bin
```

Comment:
Because /opt/chef/bin is already owned by the chef-infra package, we probably don't want to install anything here. 
Could we instead use a different directory, something like '/opt/chef/migration' or even include it in '/opt/chef/bundle"?

Reply:
Would it be a good idea to unpack the migration tool into /opt/chef/migration/bin instead?

Reply:
since new install lives in /hab why aren't we using that directory and in the end there shouldn't be any /opt/chef anything unless using omnibus version. That'll make it easier for customers to understand anything related to hab builds lives in /hab/somewhere and omnibus in /opt/chef instead of 1 tool over here and all others over there. This also then makes Cookbook checks easy to accomodate both by simply checking for the existence of /opt/chef.

Reply:
@marcparadise @Stromweld It appears that using /opt/chef or /hab may have similar drawbacks, as it could interfere with their structures. Could we use a separate directory, /opt/chef-migrate, instead, where bin would hold the binary of the migration tool and bundle would hold the tar file?

Reply:
Changed the directory to place the migration binary and bundle into `/hab/migration`. The binary is now located in `/hab/migration/bin`, and the infra tarball is stored in `/hab/migration/bundle`.